network/wifi/wifi.c

```diff
@@ -572,3 +572,25 @@ static int32_t wifi_socket_recvfrom(struct wifi_desc *desc, uint32_t sock_id,
 	/* TODO: Implement for UDP */
 	return FAILURE;
 }
+
+
+/** @brief See \ref network_interface.socket_bind */
+static int32_t wifi_socket_bind(struct wifi_desc *desc, uint32_t sock_id,
+				uint16_t port)
+{
+	return -1;
```

Comment:
ENOSYS          Function not implemented (POSIX.1-2001).

Reply:
I implemented them in the next commit, is useful to change return code?

Reply:
I am not a big fan of splitting things into multiple pull request while also splitting the functionality. A pull request should do something by itself implement a functionality or fix an issue. Different pull requests should implement dfferent things.

Reply:
It is in this PR the implementation

Reply:
Yeah, that's good point, I don't see why you would do add a stub like this and fill it in the next commit. Just skip these stubs as it pollutes git history and implement them directly in next commit.

Reply:
Squashed the commits